pom.xml

```diff
@@ -169,28 +193,25 @@
                             <artifactId>maven-jar-plugin</artifactId>
                             <version>${maven-jar-plugin.version}</version>
                         </plugin>
+
                         <plugin>
-                            <groupId>org.apache.maven.plugins</groupId>
-                            <artifactId>maven-surefire-plugin</artifactId>
-                            <version>${maven-surefire-plugin.version}</version>
-                            <dependencies>
-                                <dependency>
-                                    <groupId>org.junit.platform</groupId>
-                                    <artifactId>junit-platform-surefire-provider</artifactId>
-                                    <version>${junit-platform-surefire-provider.version}</version>
-                                </dependency>
-                            </dependencies>
+                            <groupId>org.jacoco</groupId>
+                            <artifactId>jacoco-maven-plugin</artifactId>
+                            <version>${jacoco-maven-plugin.version}</version>
                         </plugin>
                         <plugin>
                             <groupId>com.google.cloud.tools</groupId>
                             <artifactId>jib-maven-plugin</artifactId>
-                            <version>3.1.1</version>
+                            <version>${maven-jar-plugin.version}</version>
                             <configuration>
+                                <from>
+                                    <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
+                                </from>
                                 <to>
-                                    <image>169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/document-generator</image>
+                                    <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/document-generator:latest</image>
```

Comment:
why is it changed from 169942020521 to 416670754337?

Reply:
Actually in docker-composer had this so just replaced , Now changed to 169942020521